x/marker/types/msg.go

```diff
@@ -52,14 +57,12 @@ func NewMsgAddMarkerRequest(
 		SupplyFixed:            supplyFixed,
 		AllowGovernanceControl: allowGovernanceControl,
 		AllowForcedTransfer:    allowForcedTransfer,
+		RequiredAttributes:     requiredAttributes,
 	}
 }
 
 // ValidateBasic runs stateless validation checks on the message.
 func (msg MsgAddMarkerRequest) ValidateBasic() error {
-	if msg.Status == StatusUndefined {
-		return ErrInvalidMarkerStatus
-	}
```

Comment:
Why is this being removed? It seems like a good validation check to have here.

Reply:
Just seemed redundant since we have this a few lines down: 
```
	if msg.Status != StatusFinalized && msg.Status != StatusProposed {
		return fmt.Errorf("marker can only be created with a Proposed or Finalized status")
	}
```